### PR TITLE
Add copy paste tip 

### PR DESCRIPTION
**Pull request by SallyMcGrath**
## What does this change?

Module: Fundamentals
Week(s): 2

## Description

Addresses https://github.com/CodeYourFuture/fundamentals-blockly/issues/25 

tell mentors to copy and paste blocks so the trainees can chunk their solutions more effectively

## Rendered Pages

-----
[View rendered docs/fundamentals/week-2/mentors.md](https://github.com/CodeYourFuture/syllabus/blob/feature/fundamentals-blockly-issue-25/docs/fundamentals/week-2/mentors.md)

### CODE REVIEW

File: docs/fundamentals/week-2/mentors.md [hunk 1]
```diff
@@ -94,9 +101,7 @@ This is very likely to be the first time they have ever paired on a programming
 Many of the words we use will either be completely new to trainees or used in a new way. Make space in each day to define terms, and model ways to find out terms (look stuff up together)
 :::
 
-## Blockers/Coursework Review
-
-We open every session at CYF with a coursework review, where trainees identify their blockers and ask for help getting unstuck.
+We make space in every session at CYF for a coursework review, where trainees identify their blockers and ask for help _getting unstuck_.
```
Comment (gregdyke) [inline docs/fundamentals/week-2/mentors.md:106]: I wonder if keeping the heading might not be useful as a reminder?

Comment (SallyMcGrath) [reply docs/fundamentals/week-2/mentors.md:106]: Add it as a line change you monster!! (I'll accept it!)

Comment (gregdyke) [reply docs/fundamentals/week-2/mentors.md:106]:
```suggestion
## Blockers/Coursework Review

We make space in every session at CYF for a coursework review, where trainees identify their blockers and ask for help _getting unstuck_.
```

Comment (gregdyke) [reply docs/fundamentals/week-2/mentors.md:106]: I didn't know that was a thing or how to do it :)

Comment (SallyMcGrath) [reply docs/fundamentals/week-2/mentors.md:106]: Now you know! And thank you! xoxoxo You're the best, Greg.

**Comment by SallyMcGrath** [comment]
@all-contributors please add @gregdyke for infrastructure, tests, content and code

**Pull request merged**